metrics/group_workload_transactions.go

```diff
@@ -48,9 +46,9 @@ func (w *WorkloadTransactionsMetricGroup) getValidTagsKeys(scopeName string) []s
 			"priority",
 		}
 	} else {
-		fmt.Println("unknown scope name")
+		log.Error().Msg("unknown scope")
```

Comment:
You can make this function slightly simpler:

```
func (w *WorkloadTransactionsMetricGroup) getValidTagsKeys(scopeName string) []string {
	if scopeName == "default" {
		return []string{
			"env",
			"service",
			"version",
			"cluster",
		}

   if scopeName == "started" {
		return []string{
			"env",
			"service",
			"version",
			"cluster",
			"priority",
		}
	
	log.Error().Msg("unknown scope")
         return nil
	}
```

Reply:
Great catch!